src/system-fetch.js

```diff
@@ -59,6 +59,29 @@
       }
     };
   }
+  else if (typeof fetch != 'undefined') {
```

Comment:
Yes, of course. I'm just unsure what a good way to check the environment would be.

I'll move it below the require check and add a process check there.